pkg/api/testing/serialization_test.go

```diff
@@ -156,8 +156,10 @@ var nonRoundTrippableTypes = sets.NewString(
 	"WatchEvent",
 	// ListOptions is now part of the meta group
 	"ListOptions",
-	// Delete options is only read in metav1
+	// DeleteOptions, CreateOptions and UpdateOptions are only read in metav1
 	"DeleteOptions",
+	"CreateOptions",
+	"UpdateOptions",
 )
 
 var commonKinds = []string{"Status", "ListOptions", "DeleteOptions", "ExportOptions"}
```

Comment:
do CreateOptions/UpdateOptions need to be added to this list?

Reply:
I don't know, but let's do it for consistency.

Reply:
GetOptions